src/lib.rs

```diff
@@ -144,7 +144,7 @@
 //! assert_eq!(utc.to_string(), "2016-10-21 23:00:00 UTC");
 //! # }
 //! ```
-
+#![cfg_attr(not(feature="std"), no_std)]
```

Comment:
Added a linebreak underneath to make it clearer that this is unrelated to the usage of `chrono`; this is the conditional declaration that this library is `no_std`, which is required for it to be used in a `no_std` environment.

Reply:
oh I see. I believe (based on [this part of the embedded rust book](https://rust-embedded.github.io/book/intro/no-std.html#summary)) that `no_std` should be placed in the user's `lib.rs` or `main.rs`, and not in any of the libraries. We can add a test that verifies that we can in fact build in no-std environments by creating a test in `tests/` that depends on `chrono-tz` and does... something.

Reply:
`#![no_std]` triggers the availability of the core crate instead of the std crate. If it isn't present in this library, then a no_std usage of this library on a target without the standard library causes

```
error[E0463]: can't find crate for `std`
  |
  = note: the `thumbv7em-none-eabihf` target may not be installed
```

(with the architecture varying based on what target was attempted). Libraries do declare no_std, see e.g. [bme280](https://github.com/uber-foo/bme280-rs/blob/4b0652baea4b2cbe340c72efe7476a921a040600/src/lib.rs#L19) and [serde](https://github.com/serde-rs/serde/blob/95b1a5d3d9838aaf5d554658bce38385c05011ee/serde/src/lib.rs#L84).

I'm not sure how to write a test that chrono-tz builds successfully in no_std mode; I *think* that a CI can do it by specifying a target that does not support the standard library (any of the `-none-` targets would work, such as `thumbv7em-none-eabihf`) and attempting to build the crate.